docs/integration.rst

```diff
@@ -17,3 +17,39 @@ Here is an example, to add in your .pre-commit-config.yaml
     hooks:
       - id: yamllint
         args: [-c=/path/to/.yamllint]
+
+Integration with GitHub Actions
+-------------------------------
+
+Starting with version XXX of yamllint, it will automatically use the
+`annotations API <https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-a-warning-message>`
+to decorate code with lint errors automatically. It should auto-detect that it
+is running inside of `GitHub Actions<https://github.com/features/actions>` and automatically use the correct output
+format, but you can also force it with ``yamllint --format github``
+
+An example workflow using GitHub Actions
+
+.. code:: yaml
+
+   ---
+   name: yamllint test
+
+   # yamllint disable-line rule:truthy
```

Comment:
If this line is not mandatory here, can you remove it, for the clarity of the example?